src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantMandatoryQuestionsController.java

```diff
@@ -51,11 +51,32 @@ public ResponseEntity<InputStreamResource> exportSpotlightChecks(@PathVariable I
         headers.setContentDisposition(ContentDisposition.parse("attachment; filename=" + exportFileName));
 
         long end = System.currentTimeMillis();
-        log.info("Finished mandatory questions export for scheme " + schemeId + ". Export time in millis: "
+        log.info("Finished due diligence data export for scheme " + schemeId + ". Export time in millis: "
                 + (end - start));
 
         return ResponseEntity.ok().headers(headers).contentLength(length)
                 .contentType(MediaType.parseMediaType(EXPORT_CONTENT_TYPE)).body(resource);
     }
 
+    @GetMapping(value = "/spotlight-export/{schemeId}", produces = EXPORT_CONTENT_TYPE)
+    public ResponseEntity<InputStreamResource> exportSpotlightChecks(@PathVariable Integer schemeId) {
+        log.info("Started spotlight export for scheme " + schemeId);
+        long start = System.currentTimeMillis();
+
+        final ByteArrayOutputStream stream = grantMandatoryQuestionService.getSpotlightChecks(schemeId);
+        final String exportFileName = "spotlight_checks.zip";
+        final InputStreamResource resource = fileService.createTemporaryFile(stream, exportFileName);
+
+        final int length = stream.toByteArray().length;
+
+        final HttpHeaders headers = new HttpHeaders();
+        headers.setContentDisposition(ContentDisposition.parse("attachment; filename=" + exportFileName));
+
+        long end = System.currentTimeMillis();
+        log.info("Finished spotlight export for scheme " + schemeId + ". Export time in millis: " + (end - start));
+
+        return ResponseEntity.ok().headers(headers).contentLength(length)
+                .contentType(MediaType.parseMediaType(EXPORT_CONTENT_TYPE)).body(resource);
```

Comment:
Might be worth making a private method to return a file like this as there's at least 3 places which do exactly this